app/services/settings/general/upsert.rb

```diff
@@ -29,6 +37,12 @@ def self.create_tags_if_necessary(settings)
         tags = Settings::General.suggested_tags + Settings::General.sidebar_tags
         Tag.find_or_create_all_with_like_by_name(tags)
       end
+
+      def self.private_upload_logo(image)
```

Comment:
@nickytonline why do we prepend the method name with private? 
I thought we'd use ` private_class_method :upload_logo`.

Reply:
I was just going by rubocop's recommendation.

```
`private` (on line 41) does not make singleton methods private. Use `private_class_method` or `private` inside a `class << self` block instead.Lint/IneffectiveAccessModifier(RuboCop)
```

Reply:
As discussed, I'm using the codebase standard for this which is

```
      def self.upload_logo(image)
        LogoUploader.new.tap do |uploader|
          uploader.store!(image)
        end
      end

      private_class_method :upload_logo
```